procurement/tests/test_import_climate_representatives.py

```diff
@@ -0,0 +1,150 @@
+from django.test import TestCase
+
+import unittest
+from django.core.management import call_command
+
+from procurement.models import (
+    Council,
+    ClimateCouncillor,
+    ClimateRepresentative,
+    ClimateOfficer,
+)
+
+
+class ImportClimateRepresentativesTestCase(TestCase):
+    def setUp(self):
+        call_command(
+            "import_councils",
+        )
```

Comment:
This should ideally use a fixture to add the required councils rather than running the import command.